client-sdk/ts-web/rt/test/address.test.ts

```diff
@@ -0,0 +1,13 @@
+import * as oasisRT from './..';
```

Comment:
This unit test doesn't build because of this stupid jest crap and loads of transformation steps.